app/views/users/new.html.haml

```diff
@@ -2,13 +2,16 @@
   .row
     .col-sm-10.col-sm-offset-1
       = bootstrap_form_for @user, layout: :horizontal, url: register_path do |f|
+        - if @invitation
+          = hidden_field_tag :invitation_token, @invitation.token
+
         %header
           %h1 Register
         %fieldset
           = f.email_field :email, control_col: 'col-sm-6', label: 'Email Address'
```

Comment:
Alright. I've changed it.

I tend to not really like the bootstrap_form gem extension. Somehow it seems to be too much magic going on - too much abstraction away from the actual HTML.